workbench-core/example/infrastructure/scripts/cleanup.sh

```diff
@@ -85,10 +92,33 @@ function cleanup(){
             echo -e "${RED}*** [ERROR] SSM paramters could not be deleted ***${NC}"
         fi
     fi
+
+
+    if [[ -n $GET_USER_POOL_ID ]]; then
+        ### Get UserPool Id
+        echo -e "${YELLOW}### Get UserPool Id ###${NC}"
+        Id=`aws cognito-idp list-user-pools --region $REGION --max-results 5 | jq '.UserPools[] | select(.Name == "example-app-userPool") | {Id}' | jq '.Id'`
+        if [ $? -eq 0 ]; then
+        echo -e "${GREEN}Id: $Id${NC}"
+        else
+            echo -e "${RED}*** [ERROR] Could not get the UserPool Id ***${NC}"
+        fi
+    fi
+
+    if [[ -n $GET_DYNAMO_DB_TABLE ]]; then
+        ### Get DynamoDB Table
+        echo -e "${YELLOW}### Get DynamoDb Table ###${NC}"
+        DBName=`aws dynamodb list-tables --region $REGION --max-items 4 | jq '.TableNames[]' | grep Example`
+        if [ $? -eq 0 ]; then
+        echo -e "${GREEN}DBName: $DBName${NC}"
+        else
+            echo -e "${RED}*** [ERROR] Could not get the DynamoDB Table***${NC}"
+        fi
+    fi
 }
 
 function sanitize() {
-    echo 'Sanitizing environment variables..'
+    # echo -e "${BLUE}Sanitizing environment variables..${NC}"
```

Comment:
Was this commented out on purpose?

Reply:
@bhudnell yes i would like to keep it for now. It was just getting overly verbose